appveyor.yml

```diff
@@ -4,8 +4,27 @@ platform: Any CPU
 
 configuration: Release
 
-build:
-  project: src/Dynamo.All.2013.sln
-  verbosity: normal
+environment:
+  dynamo_solution: src/Dynamo.All.2013.sln
+  xmlDocs_solution: tools/XmlDocToMarkdown/XmlDocToMarkdown.sln
+  access_token:
+    secure:a53fed0916140fa078b4559a240e740871a0a67f
```

Comment:
We can't have this access token in the .yml on a public repo. This token has now been revoked. Let's talk about how to make this private.